Testscripts/Windows/PERF-MEMCACHED-BENCHMARK.ps1

```diff
@@ -0,0 +1,216 @@
+        Provision-VMsForLisa -allVMData $allVMData -installPackagesOnRoleNames "none"
+        #endregion
+
+        Write-LogInfo "Getting Active NIC Name."
+        if ($TestPlatform -eq "HyperV") {
+            $clientNicName = Get-GuestInterfaceByVSwitch $TestParams.PERF_NIC $clientVMData.RoleName `
+                $clientVMData.HypervHost $user $clientVMData.PublicIP $password $clientVMData.SSHPort
+            $serverNicName = Get-GuestInterfaceByVSwitch $TestParams.PERF_NIC $serverVMData.RoleName `
+                $serverVMData.HypervHost $user $serverVMData.PublicIP $password $serverVMData.SSHPort
+        } else {
+            $getNicCmd = ". ./utils.sh &> /dev/null && get_active_nic_name"
+            $clientNicName = (Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort `
+                -username "root" -password $password -command $getNicCmd).Trim()
+            $serverNicName = (Run-LinuxCmd -ip $serverVMData.PublicIP -port $serverVMData.SSHPort `
+                -username "root" -password $password -command $getNicCmd).Trim()
+        }
+
+        if ($serverNicName -eq $clientNicName) {
+            $nicName = $clientNicName
+        } else {
+            Throw "Server and client SRIOV NICs are not same."
+        }
+        if ($currentTestData.SetupConfig.Networking -imatch "SRIOV") {
+            $DataPath = "SRIOV"
+        } else {
+            $DataPath = "Synthetic"
+        }
+        Write-LogInfo "CLIENT $DataPath NIC: $clientNicName"
+        Write-LogInfo "SERVER $DataPath NIC: $serverNicName"
+
+        Write-LogInfo "Generating constants.sh ..."
+        $constantsFile = "$LogDir\constants.sh"
+        Set-Content -Value "#Generated by LISAv2 Automation" -Path $constantsFile
+        Add-Content -Value "server=$($serverVMData.InternalIP)" -Path $constantsFile
+        Add-Content -Value "client=$($clientVMData.InternalIP)" -Path $constantsFile
+        Add-Content -Value "nicName=$nicName" -Path $constantsFile
+        foreach ($param in $currentTestData.TestParameters.param) {
+            Add-Content -Value "$param" -Path $constantsFile
+        }
+        Write-LogInfo "constants.sh created successfully..."
+        Write-LogInfo (Get-Content -Path $constantsFile)
+
+        #region EXECUTE TEST
+        $myString = @"
+cd /root/
+./perf_memcached.sh &> MemcachedConsoleLogs.txt
+. utils.sh
+collect_VM_properties
+"@
+        Set-Content "$LogDir\StartMemcachedTest.sh" $myString
+        Copy-RemoteFiles -uploadTo $clientVMData.PublicIP -port $clientVMData.SSHPort -files "$constantsFile,$LogDir\StartMemcachedTest.sh" -username "root" -password $password -upload
+        Copy-RemoteFiles -uploadTo $clientVMData.PublicIP -port $clientVMData.SSHPort -files $currentTestData.files -username "root" -password $password -upload
+
+        Run-LinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort -username "root" -password $password -command "chmod +x *.sh" | Out-Null
```

Comment:
not "root". Use runAsSudo for the rest of lines

Reply:
Fixed.